tests/functional/run-s3-tests.sh

```diff
@@ -16,19 +16,19 @@ CH_VERSIONING=$(python -c "import oio.common.constants as cnt; print(hasattr(cnt
 RET=0
 
 # Launch test with container hierarchy with same configuration file
-for key_format in v1 v2 v3; do
-    for support_listing_versioning in false true; do
+for key_format in v2 v3; do
```

Comment:
Instead of removing tests, I suggest splitting functional tests in 2 environments: https://github.com/open-io/oio-swift/pull/209/commits/1d51d817fdad749c099277585c8765e32ddb20ae

Reply:
I don't really remember why I disabled them. I'll enable them again and we'll see...